### PR TITLE
Helmcharts for EKS deployment of the Person-Reporting-Service and Organization-Reporting-Service.

### PR DESCRIPTION
**Pull request by alog-enquizit**
NBS ELR dataflow - Option 3

Data Reporting Service has bunch of micro services organized as person, organization, investigation, observation, status, etc.. 

This Pull Request is to create Helm charts for the deployment of the following Data Reporting services:
1. Person Reporting Service
2. Organization Reporting Service

https://cdc-nbs.atlassian.net/browse/CNAT-168

### CODE REVIEW

**Review by cmoss-enq — APPROVED** [review]
Good starting point, approving and will continue testing

**Review by cmoss-enq — APPROVED** [review]
re-approving after merge with main

**Review by kramakur — APPROVED** [review]
Approved

**Pull request merged**
